back/config/locales/ar-SA.yml

```diff
@@ -145,3 +145,9 @@ ar:
       <p>تمت دعوة المبادرين لتقديم مُقترحهم في اجتماع المجلس القادم. سنقدم تحديثًا رسميًا بذلك.</p>
     default_eligibility_criteria: >
       <ul> <li> يجب أن تندرج ضمن مجالات وكفاءات السياسة المحلية</li> <li> يجب أن تخدم المصلحة العامة وليس مصلحتك الفردية</li> <li> يجب ألّا تقوم بالتمييز على أساس الجنس، العرق أو العمر أو الخلفية</li> <li>يجب ألّا تؤذي الآخرين</li> </ul>
+  reset_password_mailer:
+    send_reset_password:
+      cta_reset_password: 'إعادة تعيين كلمة المرور'
```

Comment:
I'm not sure at all, but maybe you could use Crowdin CLI to write a script to update these keys https://support.crowdin.com/cli-tool/
We talked about using it a few times but never really tried it.

It won't be easy but should bit more interesting than uploading these files manually :smile: 
Also, we could use this script in the future. Maybe better to ask Koen if it's worth the effort.

Reply:
Oh I didn't know we they have a CLI! Yes, uploading the translation files on the command line is easier than doing it via their Web UI, thanks for the tip!